### DISCUSSION TITLE
2023/07/28/enforce-conventions-but-dont-go-overboard-with-architecture-tests/

### DISCUSSION THREAD

**Discussion by giscus** [CodingMilitia/Blog#94] [Blog comments]
# 2023/07/28/enforce-conventions-but-dont-go-overboard-with-architecture-tests/

I’ve been trying to flee the over-engineered C# solution structures, with multiple projects, with related code scattered around. Even if an imperfect solution, I’ve turned to NetArchTest to help in guide on following some conventions in a simplified solution structure.

https://blog.codingmilitia.com/2023/07/28/enforce-conventions-but-dont-go-overboard-with-architecture-tests/

**Comment by marcingolenia** [comment]
Hi there! 
Great post, thank You for sharing. One question though; Why API is not the part of a slice?

**Reply by joaofbantunes** [reply]
Hey! Not really sure what you mean.
